test/OpenTelemetry.Contrib.Shared.Tests/SqlProcessorTestCases.cs

```diff
@@ -21,12 +21,50 @@ public static class SqlProcessorTestCases
 
     public static TheoryData<TestCase> GetSemanticConventionsTestCases()
     {
+        var data = new TheoryData<TestCase>();
+
         var assembly = Assembly.GetExecutingAssembly();
         var input = JsonSerializer.Deserialize<TestCase[]>(
             assembly.GetManifestResourceStream("SqlProcessorTestCases.json")!,
             JsonSerializerOptions)!;
 
-        var data = new TheoryData<TestCase>();
+        if (input is not null)
+        {
+            foreach (var testCase in input)
+            {
+                // Skipping this for now as we need to discuss the expected output.
+                if (testCase.Name == "alter_table")
+                {
+                    continue;
+                }
+
+                if (DbSystemTestCasesToExecute.Contains(testCase.Input.DbSystemName))
+                {
+                    data.Add(testCase);
+                }
+            }
+        }
+
+        // Reintroduce "alter_table" with a revised expected output.
+        // We don't expect the number '255' to be redacted when defining the size of a column.
+        data.Add(new()
```

Comment:
@alanwest This specific test in the semantic conventions expects the `255` literal to be sanitized. My current implementation handles column definitions such that we retain the value. If you're happy with this, should I update this in the semantic conventions?